File: tests/fixtures/code_samples/java/sample_037.java
// Sample 37: small utility.
package samples;

import java.util.List;

public final class Sample037 {
    private Sample037() {}

    public static int operation(List<Integer> xs) {
        int total = 37;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 37) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_037.py
"""Sample 37: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_37(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 37."""
    total = 37
    for x in xs:
        total += int(x)
    return total


def operation_37_pure(value: int) -> int:
    return (value * 37) % 7919

